lib/flags.rb

```diff
@@ -0,0 +1,819 @@
+class CustomWizard::Flag
+    def initialize(id, name, value)
+        @id     = id
+        @name   = name
+        @value = value
+    end
+
+    def id
+        @id
+    end
+
+    def name
+        @name
+    end
+
+    def value
+        @value
+    end
+end
+
+class CustomWizard::Flags
+
+    def self.list
```

Comment:
@merefield We need to put the country_code: flag list into a .yml file. That's the standard way to store static data in Discourse. You can follow the format / copy the code from the locations plugin countries library: https://github.com/paviliondev/discourse-locations/blob/master/lib/country.rb#L2

Reply:
This is done, pending commit and feels much ... nicer! :)